controllers/storagecluster/storagecluster_controller.go

```diff
@@ -188,6 +189,36 @@ func (r *StorageClusterReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		},
 	)
 
+	storageConsumerPredicate := predicate.Funcs{
```

Comment:
Let's have the name describe the condition itself and not the structure that holds it  
```suggestion
	ocsClientOperatorVersionPredicate := predicate.Funcs{
```

Reply:
done.